packages/bruno-app/src/providers/ReduxStore/slices/collections/actions.js

```
@@ -193,6 +194,62 @@ export const saveRequest = (itemUid, collectionUid, silent = false) => (dispatch
+        if (!silent) {
+          toast.success('File saved successfully!');
+        }
+      })
+      .then(resolve)
+      .catch((err) => {
+        if (!silent) {
+          toast.error('Failed to save file!');
+        }
+        reject(err);
+      });
+  });
```

Biome (2.4.16)

[error] Promise executor functions should not be `async`.

(lint/suspicious/noAsyncPromiseExecutor)